Design.md

```diff
@@ -0,0 +1,52 @@
+# Design
+ 
+## Senior UI/UX Designer
+- I have at least 5 years experience
+- I display initiative and ownership
+
+- I can lead a project day to day (Design Lead)
+    - I can make strategic design and UX decisions about both core and new functions and features
+    - I can measure and capture user feedback on a regular basis.
+- I can create design guidelines to set the tone and design rules through out a project life-cycle
+- I can oversee the user experience from conception to launch in partnership with product owners and developers
+- I am responsible for overall team performance and building a culture of design excellence
+- I am comfortable with running the Scrum ceremonies (i.e. can act as Scrum Master)
+- I am comfortable with mentoring/teaching other designers 
+- I can define the user experience best practice for client porjects and internal projects
+
+
+##  UI/UX Designer
+- I have at least 3 years experience
+- I have Excellent visual design skills with an eye for detail
+
+- I can build/gather requirements from client conversations
+    - I can assist on Specification Reviews
+    - I can do initial meetings with clients
+    - I can creates the design of user interface specifications, by applying client research and design guidelines
+- I can design user interfaces that encompass an end-to-end experience across a variety of platforms
+- I can represent work to clients and other team members, clearly and succinctly articulating the goals and design concepts
+- I can excute the UX process from end to end, including wireframes, behavioral specifications, and high-fidelity prototypes
```

Comment:
1. "end-to-end" should have hyphen as per https://en.wikipedia.org/wiki/End-to-end_principle

Reply:
Done